.github/labeler.yml

```
@@ -12,13 +12,19 @@ wallet-dashboard:
       - any-glob-to-any-file: apps/wallet-dashboard/**/*
 ts-sdk:
   - changed-files:
-      - any-glob-to-any-file: sdk/**/*
+      - any-glob-to-any-file:
```

```suggestion
      - any-glob-to-any-file: |
```
the list below has this, not sure if needed